src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -142,6 +206,9 @@ def evaluate(
         )
 
     input_data_df = pd.read_json(data, lines=True)
+    if tempfile_created:
+        # During the run we have created the temprary file. We will delete it here.
+        os.unlink(data)
     input_data_df = input_data_df.rename(columns={col: f"inputs.{col}" for col in input_data_df.columns})
 
     result_df = pd.concat([input_data_df, evaluators_result_df], axis=1, verify_integrity=True)

```

Comment:
FYI - Just had a sync with @singankit. The target output needs to be presented in the final output of evaluate api.

The final output will contain below three parts:
a. data
	i. Inputs.col1
	ii. Inputs.col2
b. target
	i. Outputs.col1
	ii. Outputs.col2
c. evaluators
        i. Outputs.<evaluator name>.col1

Reply:
Renamed the columns from target.